src/components/DetailsPanel/index.js

```diff
@@ -43,7 +44,12 @@ const DetailsPanel = ({ navigation, collection, savedRecordsGroupedByType }) =>
           />
         );
       case TIME_SAVED:
-        return <Text>TimeSaved</Text>;
+        return (
+          <TimeSavedAccordionsContainer
+            isDescending={sortingState.sortDirections[TIME_SAVED] === SORT_DESC}
```

Comment:
`TimeSavedAccordionsContainer` no longer handles `isDescending`

(also: `DateAccordionsContainer` & `SubTypeAccordionsContainer`)

![image](https://user-images.githubusercontent.com/3383704/116619261-37754280-a90e-11eb-83bf-fae536b927af.png)

Reply:
`isDescending` in `SubTypeAccordionsContainer` was indeed leftover. The other 2 instances were used in `ownProps`. But given that each selector is only used for those specific components, I refactored to extract `isDescending` in the selector itself.